WooCommerce/Classes/Authentication/Store Creation/StoreCreationSummaryView.swift

```diff
@@ -0,0 +1,153 @@
+import SwiftUI
+
+/// Hosting controller that wraps the `StoreCreationSummaryView`.
+final class StoreCreationSummaryHostingController: UIHostingController<StoreCreationSummaryView> {
+    private let onContinueToPayment: () -> Void
+
+    init(viewModel: StoreCreationSummaryViewModel,
+         onContinueToPayment: @escaping () -> Void) {
+        self.onContinueToPayment = onContinueToPayment
+        super.init(rootView: StoreCreationSummaryView(viewModel: viewModel))
+
+        rootView.onContinueToPayment = { [weak self] in
+            self?.onContinueToPayment()
+        }
+    }
+
+    required dynamic init?(coder aDecoder: NSCoder) {
```

Comment:
Nit: we can also add an `@available` attribute here to signal the compiler in compile-time that this initializer is not ready for use:

```suggestion
    @available(*, unavailable)
    required dynamic init?(coder aDecoder: NSCoder) {
```

Reply:
Nice, updated in https://github.com/woocommerce/woocommerce-ios/pull/8109/commits/b0b8bf98192e47653259d2e9e048aeabdc2f8130